operator/config/rbac/role.yaml

```diff
@@ -6,6 +6,42 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - logging.banzaicloud.io
+  resources:
+  - clusterflows
+  - clusteroutputs
+  - flows
+  - outputs
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - logging.banzaicloud.io
+  resources:
+  - clusterflows/status
+  - flows/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+  - logging.banzaicloud.io
+  resources:
+  - output
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
```

Comment:
why is output listed twice?

Reply:
this was created by kubebuilder I will see why it's doing that